release notes/v0.48.0.md

```diff
@@ -27,6 +27,21 @@ Allows users to throttle the CPU from chrome/chromium's perspective, which helps
 ...
 ```
 
+### Add support for browser module's `page.throttleNetwork` [browser#887](https://github.com/grafana/xk6-browser/issues/887)
```

Comment:
Should we link PR https://github.com/grafana/xk6-browser/pull/1094 instead?

Reply:
Nice catch, thanks! Fixed in 928d2b3f44594c4be130e4f1808b5ff22e2fc32b